spec/factories/compliment_factory.rb

```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :compliment do
+    message 'You rock!'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.